core/domain/entities/editor/blocks/CoreBlocksAssetManager.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace EventEspresso\core\domain\entities\editor\blocks;
+
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidEntityException;
+use EventEspresso\core\services\assets\BlockAssetManager;
+use EventEspresso\core\services\collections\DuplicateCollectionIdentifierException;
+
+/**
+ * Class CoreBlocksAssetManager
+ * Manages Editor Block Assets for Event Espresso Core
+ *
+ * @package EventEspresso\core\domain\entities\editor\blocks
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class CoreBlocksAssetManager extends BlockAssetManager
```

Comment:
It may have been explained elsewhere already and if so my apologies, I've forgotten.  What exactly are `Core Blocks` again?

Reply:
Core Blocks are Blocks for Core of course ! :laughing:
But seriously this is the `BlockAssetManager` for the EE core JS,
as opposed to something like `RecurringEventsBlocksAssetManager`.
This could maybe be broken up into more granular managers like `AdminBlockAssetManager` for admin-only assets, and `PublicBlockAssetManager` for frontend-only assets, but I *think* everything in here is shared between the frontend and admin.